valhalla/mjolnir/landmark_database_builder.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <boost/property_tree/ptree.hpp>
+#include <cstdint>
+#include <iostream>
+#include <list>
+#include <string>
+#include <vector>
+
+#include "baldr/graphconstants.h"
+#include "filesystem.h"
+#include "mjolnir/util.h"
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/geometries.hpp>
+#include <boost/geometry/geometries/register/point.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include <geos_c.h>
+
+namespace valhalla {
+namespace mjolnir {
+struct LandmarkDatabase {
+public:
+  LandmarkDatabase(std::string& db_name) : db(nullptr), database(db_name) {}
+  bool openDatabase();
```

Comment:
nit: generally we use snake case for everything to kind of match the stl. there are some parts of the code that match the google style like you have here but havent been doing that for quite some time except in the case where there is already an established class. for new stuff snake case please